src/main/java/com/foundation/search/model/Criteria.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright (c) 2018 Jalasoft.
+ * 2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jalasoft, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
+package com.foundation.search.model;
+
+public class Criteria {
+    public String path;
```

Comment:
Please change the access modifier to private instead of public. These attributes are going to be accessible only by its Setters and Getters.